Currently Loaded Modulefiles:
 1) slurm/slurm/23.02.4   2) nvidia-hpc/2024_241   3) nvhpc/24.1  
no change     /cm/shared/apps/anaconda/2024.02/condabin/conda
no change     /cm/shared/apps/anaconda/2024.02/bin/conda
no change     /cm/shared/apps/anaconda/2024.02/bin/conda-env
no change     /cm/shared/apps/anaconda/2024.02/bin/activate
no change     /cm/shared/apps/anaconda/2024.02/bin/deactivate
no change     /cm/shared/apps/anaconda/2024.02/etc/profile.d/conda.sh
no change     /cm/shared/apps/anaconda/2024.02/etc/fish/conf.d/conda.fish
no change     /cm/shared/apps/anaconda/2024.02/shell/condabin/Conda.psm1
no change     /cm/shared/apps/anaconda/2024.02/shell/condabin/conda-hook.ps1
no change     /cm/shared/apps/anaconda/2024.02/lib/python3.11/site-packages/xontrib/conda.xsh
no change     /cm/shared/apps/anaconda/2024.02/etc/profile.d/conda.csh
no change     /home/qmao.cs11/.bashrc
No action taken.

CondaError: Run 'conda init' before 'conda activate'

srun: error: DGX-CN02: task 0: Exited with exit code 1
/home/qmao.cs11/.conda/envs/RLR/lib/python3.10/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
Loading LLM
Loading checkpoint shards:   0%|          | 0/3 [00:00<?, ?it/s]Loading checkpoint shards:  33%|███▎      | 1/3 [00:01<00:02,  1.12s/it]Loading checkpoint shards:  67%|██████▋   | 2/3 [00:02<00:01,  1.09s/it]Loading checkpoint shards: 100%|██████████| 3/3 [00:02<00:00,  1.05it/s]Loading checkpoint shards: 100%|██████████| 3/3 [00:02<00:00,  1.01it/s]
